src/Platforms/Keywords/MySQL84Keywords.php

```diff
@@ -30,12 +33,44 @@ public function getName()
     /**
      * {@inheritDoc}
      *
-     * @link https://dev.mysql.com/doc/refman/8.4/en/keywords.html#keywords-new-in-current-series
+     * @link https://dev.mysql.com/doc/refman/8.4/en/keywords.html
      */
     protected function getKeywords()
     {
         $keywords = parent::getKeywords();
 
+        // Removed Keywords and Reserved Words
+        $keywords = array_diff($keywords, [
+            'GET_MASTER_PUBLIC_KEY',
+            'MASTER_AUTO_POSITION',
+            'MASTER_BIND',
+            'MASTER_COMPRESSION_ALGORITHMS',
+            'MASTER_CONNECT_RETRY',
+            'MASTER_DELAY',
+            'MASTER_HEARTBEAT_PERIOD',
+            'MASTER_HOST',
+            'MASTER_LOG_FILE',
+            'MASTER_LOG_POS',
+            'MASTER_PASSWORD',
+            'MASTER_PORT',
+            'MASTER_PUBLIC_KEY_PATH',
+            'MASTER_RETRY_COUNT',
+            'MASTER_SSL',
+            'MASTER_SSL_CA',
+            'MASTER_SSL_CAPATH',
+            'MASTER_SSL_CERT',
+            'MASTER_SSL_CIPHER',
+            'MASTER_SSL_CRL',
+            'MASTER_SSL_CRLPATH',
+            'MASTER_SSL_KEY',
+            'MASTER_SSL_VERIFY_SERVER_CERT',
+            'MASTER_TLS_CIPHERSUITES',
+            'MASTER_TLS_VERSION',
+            'MASTER_USER',
+            'MASTER_ZSTD_COMPRESSION_LEVEL',
```

Comment:
The only keywords that are implemented in the parent keyword classes are `MASTER_BIND` and `MASTER_SSL_VERIFY_SERVER_CERT`. We currently don't need all the other ones.

Please also check the tests and create one that covers your changes.